src/pages/taxonomy/term/sort/entities/[id].tsx

```diff
@@ -43,6 +43,7 @@ export const allCategories = [
   TopicCategoryType.events,
   TopicCategoryCustomType.subterms,
   TopicCategoryCustomType.exercisesContent,
+  TopicCategoryType.exercises,
```

Comment:
![grafik](https://user-images.githubusercontent.com/13507950/215786191-63d42faf-8adb-426a-93a5-918a91d3e68e.png)

"Das passt zu mir" is a `folder`, but because it's nested, we don't want to sort them (they shouldn't appear on the higher level anymay. To sort them, you need to go to "Gesellschaftliches Engagement", so it is fine - I can add this as a comment

Reply:
Thanks for your reply, I understand now.
Better than a comment it would be to find a more precise/descriptive name for the variable (allSupportedCategories, firstLevelCategories, I don't know...), but I think it is good enough so.